src/components/source_editor/Lattice.js

```diff
@@ -7,6 +7,8 @@ import ExpansionPanel, {ExpansionPanelSummary, ExpansionPanelDetails,} from 'mat
 
 import LatticeConfigurationDialog from './LatticeConfigurationDialog';
 
+export const nonPeriodicLatticeScalingFactor = 2.0;
```

Comment:
local implementation of lattice scaling factor since we cannot pull it from `web-app`

Reply:
Same as https://github.com/Exabyte-io/exabyte-web-app/pull/1665#discussion_r759749189. Should be defined in and reused from made.js.